net-misc/icingadb/metadata.xml

```diff
@@ -13,6 +13,9 @@
 		<email>proxy-maint@gentoo.org</email>
 		<name>Proxy Maintainers</name>
 	</maintainer>
+	<use>
+		<flag name="migrate">Install the icingadb-migrate tool.</flag>
```

Comment:
Nitpick but we (proxy-maint) generally don't use `.` at the end of a descriptive text like this:
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Descriptive_text